scipy/stats/stats.py

```diff
@@ -4881,20 +5099,78 @@ def ks_2samp(data1, data2):
     data2 = np.sort(data2)
     n1 = data1.shape[0]
     n2 = data2.shape[0]
+    if min(n1, n2) == 0:
+        return Ks_2sampResult(1.0, 1.0)
```

Comment:
Does this make sense? Or raise an Error?

Reply:
Changed to raise a ValueError.